src/components/floating-bubble/floating-bubble.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faComments } from '@fortawesome/free-solid-svg-icons';
+
+const FloatingBubble = () => {
+  const handleBubbleClick = () => {
+    location.href = 'https://rulesgpt.ssw.com.au/';
```

Comment:
The component sounds generic yet the redirect location is hard-coded. Is this the intended behaviour?

Reply:
Done - I removed the hardcoding and added a prop for passing the link :)